distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -85,14 +85,21 @@ To set multi-valued attributes, use a separate override for each value. For exam
 `*topic.keyword=PPI*` +
 `*topic.keyword=radiology*`
 
-Attributes will only be overridden if they are part of the <<_metacard_type, metacard type>>, however the attributes do not have to be present on the metacard to be set.
+Attributes will only be overridden if they are part of the <<_metacard_type, metacard type>> or are <<_attribute_injection_definition,injected>>.
+
+All attributes in the <<_catalog_taxonomy, catalog taxonomy tables>> are injected into all metacards by default and can be overridden.
+
+[IMPORTANT]
+====
+If an overriden attribute is not part of the <<_metacard_type, metacard type>> or <<_attribute_injection_definition,injected>> the attribute will not be added to the metacard.
```

Comment:
✏️ overriden should be overridden

Reply:
Doh! Should have waited a little longer before merging.